Packs/Cybereason/Integrations/Cybereason/Cybereason.yml

```diff
@@ -362,6 +362,19 @@ script:
     - contextPath: Cybereason.Malops.Status
       description: Malop managemant status.
       type: string
+  - arguments:
+    - description: malopGuid of the Cybereason Malop.
+      name: malopGuid
+      required: true
+    description: Get Malop Details.
```

Comment:
Update description to "Get Management Malop details"

Reply:
Updated